_includes/markdown/Structure.md

```diff
@@ -197,3 +197,51 @@ indent_size = 4
 ```
 
 Developers may extend and/or customize these rules as new file formats are added to the project.
+
+<h2 id="privacy">Privacy {% include Util/top %}</h2>
+
+Software and services should be designed with privacy in mind, both for user trust and for potential legal reasons. The following are some general guidelines and WordPress-focused features and tactics to use. Accommodations for specific regulations such as GDPR should be determined on a case-by-case basis.
+
+Broadly, the following areas should be considered when designing for privacy:
+
+- What user data is collected and how it is used, including telemetry and reporting.
+- Where and what user data is exposed (e.g. via REST API).
+- Compliance requirements for specific user groups.
+- The content and display of a privacy policy.
+- Third-party services and advertisements.
+
+### WordPress features
+
+Since 4.9.6, WordPress provides direct support for exporting user data, removing user data, and adding a privacy policy. These do not automatically make a site compliant with a set of regulations; procedures still need to be established for response and removal timelines as well as general consideration of how data is collected and used in the first place.
+
+Another feature to keep an eye on is the REST API, in particular any meta added to the user endpoints. Be careful that exposing meta via an endpoint for the purposes of an admin feature does not inadvertently expose it publicly or to other non-admin users.
+
+### Privacy Development Best Practices
+
+#### Projects should document user data that is collected and stored.
+
+What data is collected and what is stored? How and where is the data stored? How and why is the data being collected? Where did the data come from?
+
+#### Data capture should be secure and respect individual rights.
+
+Consent must be given before the collection of personal data, both for the type of data and the purpose. Make sure consent is clear and not opted-in by default.
```

Comment:
Does this include IP address/location? If so, would we want to do this outside of a GDPR compliant website?

Reply:
I don't think the actual regulation matters so much as this being what "designing for privacy" means. And yes, it would include IP address and location.

Reply:
I think the issue here has to do with what we consider to be "personal" data. It is not, for example, my view (or I think an objective reading of GDPR's view) that stock data collected by Google Analytics (that's not augmented programmatically) does not constitute anything like personal information or require disclosure. That said, many implementing GDPR are opting for a _much_ more cautious approach, and IMO, creating a terrible UX that requires an opt in even for basic and effectively anonymized analytic data (or cases where data may not anonymized - i.e. ad trackers - but _we're_ not collecting new data) that's especially confusing and toxic for non sophisticated visitors. Meaning, this directive in our best practices is rather vague in terms of understanding implementation guidance.

My view is that we should create an alternative practice that is similar to this, but makes it clear that by personal data we mean associated clearly identifiable information about the visitor including collection of name, DOB, phone number, email address (etc) in a contact form or by some other third party tracker.

Reply:
updated the language slightly here in https://github.com/10up/Engineering-Best-Practices/pull/229/commits/550e6c0a0ac6a7cdbca05d1b30668492f9b63309